CONTRIBUTING.md

```diff
@@ -32,14 +24,18 @@ Comment modifier ces textes ? Rendez-vous dans la section ["🏗️ En pratique"
   - [Si vous n'avez que ⌛ 2 minutes pour faire une remarque](#si-vous-navez-que--2-minutes-pour-faire-une-remarque)
   - [Si vous voulez participer à l'amélioration du modèle \[⌛ 20 minutes\]](#si-vous-voulez-participer-à-lamélioration-du-modèle--20-minutes)
   - [Si vous voulez construire vous-même de nouveaux modèles \[⌛ 1 heure ou 2\]](#si-vous-voulez-construire-vous-même-de-nouveaux-modèles--1-heure-ou-2)
-  - [Vous voulez contribuer comme un pro](#vous-voulez-contribuer-comme-un-pro)
-    - [Sans rien installer sur votre machine \[⌛ \< 15 minutes de mise en place\]](#sans-rien-installer-sur-votre-machine---15-minutes-de-mise-en-place)
-    - [En adoptant les outils et pratiques des développeurs](#en-adoptant-les-outils-et-pratiques-des-développeurs)
+  - [Si vous voulez contribuer comme un pro](#si-vous-voulez-contribuer-comme-un-pro)
 
 <!-- vim-markdown-toc -->
 
 ---
 
+## ✒️ Les textes
+
+Sans rentrer dans le calcul, il y a plein de textes à améliorer ! Un exemple facile pour commencer : voici [le texte de la question sur la surface du logement](https://github.com/incubateur-ademe/nosgestesclimat/blob/master/data/logement/logement.publicodes#L49). Autre exemple : [le texte d'aide de la question "Votre logement est-il un appartement ?" ](https://github.com/incubateur-ademe/nosgestesclimat/blob/master/data/logement/logement.publicodes#L81).
+
+Comment modifier ces textes ? Rendez-vous dans la section ["🏗️ En pratique" ci-dessous](https://github.com/incubateur-ademe/nosgestesclimat/blob/master/CONTRIBUTING.md#%EF%B8%8F-en-pratique).
+
```

Comment:
On a vraiment envie de laisser ça ? Les gens peuvent faire des retours mais je ne les enjoindrais pas à nous proposer des relectures.

Reply:
Bonne question ..! T'en penses quoi ?

Reply:
J'aurais enlevé ces deux paragraphes. Ça fait un peu trop "work in progress" et ça prend de la place dans la doc pour pas grand chose. Je pense que si quelqu'un est choqué d'un truc qu'on a écrit il nous envoie un retour, naturellement.